docs/data-sources/release_policy.md

```diff
@@ -0,0 +1,88 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "launchdarkly_release_policy Data Source - launchdarkly"
+subcategory: ""
+description: |-
+  Provides a LaunchDarkly release policy data source.
+  This data source allows you to retrieve release policy information from your LaunchDarkly project.
```

Comment:
Is this generated from the description on line 14 in the Go code? This doesn't match -- does it need to be regenerated?